release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -898,6 +898,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|cert-manager Operator
+|General Availability
+|General Availability
+|General Availability
```

Comment:
Title is "Technology Preview tracker", but for cert-manager, all columns in https://61494--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-technology-preview show GA. Is it good to put such all-column-GA entry under a table called "Technology Preview tracker"?

Reply:
Similar view, +1.

Reply:
@xingxingxia @swghosh Updated the PR. Added the section instead of entry in a table